lib/api.rb

```diff
@@ -59,6 +100,12 @@ def validation_error(msg)
     if !js.strip.blank?
       params.merge!(JSON.parse(js))
     end
+  ## needed for the web vault, which doesn't use the content-type
+  elsif request.accept.to_s.match(/application\/json/) && !request.content_type.to_s.match(/application\/x-www-form-urlencoded/)
+    js = request.body.read.to_s
+    if !js.strip.blank?
+      params.merge!(JSON.parse(js))
```

Comment:
i don't think this is actually needed. Even though I haven't done an extensive trial of all options in the webvault, but adding and importing of keys does not need this.

Reply:
It’s possible that it is not needed anymore